docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3"
+
+services:
+  db:
+    image: postgres:13
+    env_file:
+      # Use template env file ./docker/secrets.env.sample
+      #   and generate your container secrets
+      - ./docker/secrets.env
+    environment:
+      POSTGRES_DB: anemperfeina
+      POSTGRES_USER: anemperfeina
+
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  app:
+    build:
+      context: .
+    image: anemperfeina:1.0
+    depends_on:
+      - db
+    ports:
+      - 8000:8000
+
+    environment:
```

Comment:
Composition envs should be also loaded here to ensure that same values are passed to both services

```
    env_file:
      - ./docker/secrets.env
```